kibana/versions.yaml

```diff
@@ -1,6 +1,16 @@
+{% if(salt['pillar.get']('kibana:repoVersion')) %}
+  {% set versionRepo = salt['pillar.get']('kibana:repoVersion') %}
+  {% if versionRepo.startswith('5.') %}
+{{ versionRepo }}:
+  repo_url: https://artifacts.elastic.co/packages/5.x/apt
```

Comment:
I wonder if there is a way the formula can access the data under dict key `5` while still setting your full version number 5.5.1. Clearly, the repo for all versions of ES 5 is the same.

Reply:
I think the version 5 thing will work out so far, but Elastic will probably mess this up again and then we'll have to cover yet another case. The good thing about the way it's implemented now is that it works for older and current versions alike.

We should probable put in a memo somewhere so that when future versions get released, anyone interested in maintaining knows what's up.

Reply:
@blbradley The only way to fix this would be to do some magic in the map.jinja where the repo dict is used. I don't believe I am good enough to get this done in a short amount of time, so this PR can go through.

I created a issue regarding that #17

Reply:
I agree! This magic you created will work.